ts/parts/Utilities.ts

```diff
@@ -1314,6 +1314,19 @@ H.merge = function<T> (): T {
     return ret;
 };
 
+/**
+ * Constrain a value to within a lower and upper threshold.
+ *
+ * @private
+ * @param {number} value The initial value
+ * @param {number} min The lower threshold
+ * @param {number} max The upper threshold
+ * @return {number} Returns a number value within min and max.
+ */
+function clamp(value: number, min: number, max: number): number {
```

Comment:
This is a quite frequent operation in Highcharts. Now that we have a utility function for that, we should at the same time identify all other places where it is done, and implement the utility function. Search for places with nested `Math.min` and `Math.max` calls.

Reply:
Still an issue.